### PR TITLE
Add StudentGrades component with styles and place it into StudentBox …

### PR DESCRIPTION
**Pull request by kamilstan**
…component

### CODE REVIEW

File: src/components/StudentGrades/StudentGrades.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                style={{ color: '#E02735' }}
+                fontSize="inherit"
+              />
+            }
+          />
+        </div>
+      </div>
+    </Box>
+  );
+};
```
Comment (patrycja-dev41171) [inline src/components/StudentGrades/StudentGrades.tsx:181]: Good structure of component.

File: src/components/StudentGrades/StudentGrades.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.student-grades__typography {
+    display: flex;
+    align-items: flex-end;
+    text-align: left;
+    height: 41px;
+}
+.student-grades__rating {
+    display: flex;
+    justify-content: space-around;
+}
```
Comment (patrycja-dev41171) [inline src/components/StudentGrades/StudentGrades.css:28]: Good use of classes.

**Review by patrycja-dev41171 — COMMENTED** [review]
Good code.

File: src/components/StudentGrades/StudentGrades.tsx (hunk 1 above)
Comment (MiroslawChmielecki) [reply src/components/StudentGrades/StudentGrades.tsx:181]: For me good looking clean code 👍

**Pull request merged**